app/models/authenticator/ldap.rb

```diff
@@ -12,11 +12,10 @@ def lookup_by_identity(username)
     private
 
     def ldap
-      @ldap ||= ldap_bind(config[:bind_dn], config[:bind_pwd])
+      @ldap = MiqLdap.new(:auth => config)
     end
 
     def ldap_bind(username, password)
-      ldap = MiqLdap.new(:auth => config)
```

Comment:
this now creates 2 different `MiqLdap` instances. Especially since `ldap` has been changed from `||=` to `=`
Was this intentional?